clang/tools/scan-build-py/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ foreach(lib ${LibScanbuild})
                      DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/lib/libscanbuild/${lib})
   list(APPEND Depends ${CMAKE_BINARY_DIR}/lib/libscanbuild/${lib})
   install(FILES lib/libscanbuild/${lib}
-          DESTINATION lib${CLANG_LIBDIR_SUFFIX}/libscanbuild
+          DESTINATION lib/libscanbuild
```

Comment:
https://github.com/llvm/llvm-project/blob/main/clang/tools/scan-build-py/bin/intercept-build#L11

so we're looking into `../lib`, I agree with the patch, LGTM